region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -361,66 +395,6 @@
         "Freeze the final Waver high and walljump up, or wait for the second Waver to come back."
       ]
     },
-    {
-      "link": [2, 1],
-      "name": "Ice and HiJump",
-      "requires": [
-        "canUseFrozenEnemies",
-        "HiJump",
-        "canCarefulJump",
-        {"or": [
-          "canTrickyJump",
-          "canWalljump"
-        ]},
-        {"spikeHits": 3},
-        {"or": [
-          "canIframeSpikeJump",
-          {"spikeHits": 3}
-        ]}
-      ],
-      "note": "Use the Fireflea when it moves to the right to cross the first gap."
-    },
-    {
-      "link": [2, 1],
-      "name": "X-Ray Access IBJ",
-      "requires": [
-        "h_canJumpIntoIBJ",
-        {"spikeHits": 5},
-        {"or": [
-          "canIframeSpikeJump",
-          {"spikeHits": 3}
-        ]}
-      ]
-    },
-    {
-      "link": [2, 1],
-      "name": "X-Ray Access Springwall",
-      "requires": [
-        "canSpringwall",
-        {"spikeHits": 5},
-        {"or": [
-          "canIframeSpikeJump",
-          {"spikeHits": 3}
-        ]}
-      ]
-    },
-    {
-      "link": [2, 1],
-      "name": "X-Ray Access Speed HiJump",
-      "requires": [
-        "HiJump",
-        {"spikeHits": 4},
-        "SpeedBooster",
-        {"or": [
-          "canTrickyJump",
-          "canWalljump"
-        ]},
-        {"or": [
-          "canIframeSpikeJump",
-          {"spikeHits": 2}
-        ]}
-      ]
-    },
     {
       "link": [2, 1],
       "name": "X-Ray Access SpikeJumps with HiJump",
```

Comment:
This strat can be split to use 3 and 4.
`spikeHits:3` = [2,3] spikehits:1 + [3,4] spikehits:2
`or#1` = [2,3]
`or#2` = [4,1]
And the [4,1] part would be notable.  The reset isn't as important.

Reply:
The [4, 1] or has
- A pixel precise positioning with moonwalk/xray/morph turnaround + insane jump +  insanewalljump
- A spike damage boost + iframeSpikeRun + TrickyJump + ~precise walljump